tests/analytics/test_events.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+
+from great_expectations.analytics.actions import (
+    EXPECTATION_SUITE_CREATED,
+    EXPECTATION_SUITE_DELETED,
+    EXPECTATION_SUITE_EXPECTATION_CREATED,
+    EXPECTATION_SUITE_EXPECTATION_DELETED,
+    EXPECTATION_SUITE_EXPECTATION_UPDATED,
+)
+from great_expectations.analytics.events import (
+    DataContextInitializedEvent,
+    ExpectationSuiteCreatedEvent,
+    ExpectationSuiteDeletedEvent,
+    ExpectationSuiteExpectationCreatedEvent,
+    ExpectationSuiteExpectationDeletedEvent,
+    ExpectationSuiteExpectationUpdatedEvent,
+)
+
+
+@pytest.mark.parametrize(
+    "event_cls, input_kwargs, expected_properties",
+    [
+        pytest.param(
```

Comment:
Pretty silly tests but I want to be extremely careful that we're not changing payload shapes down the line - these are defensive and cost very little